tests/TestEntry.cpp

```diff
@@ -156,6 +158,9 @@ void TestEntry::testResolveUrl()
     QCOMPARE(entry->resolveUrl(""), QString(""));
     QCOMPARE(entry->resolveUrl(testUrl), "https://" + testUrl);
     QCOMPARE(entry->resolveUrl("http://" + testUrl), "http://" + testUrl);
+    // Test file:// URL's
+    QCOMPARE(entry->resolveUrl("file://" + testFileUnix), "file://" + testFileUnix);
+    QCOMPARE(entry->resolveUrl("file:///" + testFileWindows), "file:///" + testFileWindows);
```

Comment:
Here is my issue with this. If you didn't add the "file:///" at this function call, then the function "resolveUrl" would prepend an "https://" to the file location (ie, "https://c:/WINDOWS/test.txt") which is obviously not desired. I think we need to refactor the resolveUrl function a little.

Reply:
You mean prepending the schema at `resolveUrl()` if the url starts with a slash or `[a-Z]:/`?

Reply:
Yes, that would be an acceptable option. The regex would be ```([a-Z]:)?\/```

Reply:
@droidmonkey fixed